src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/wataskconfigurationapi/controllers/PostConfigureTaskTest.java

```diff
@@ -32,18 +33,18 @@ public void cleanUp() {
     @Test
     public void given_configure_task_then_expect_task_state_is_assigned() throws Exception {
         caseId = createCcdCase();
-        createTaskMessage = createBasicMessageForTask()
-            .withCaseId(caseId)
-            .build();
-        taskId = createTask(createTaskMessage);
-        log.info("task found [{}]", taskId);
+
+        String taskTypeId = "followUpOverdueReasonsForAppeal";
+        createTaskMessage = createBasicMessageForTask(taskTypeId, caseId).build();
+        this.taskId = createTask(createTaskMessage);
```

Comment:
I don't think the keyword `this.` is necessary

Reply:
Those changes belong to David